tests/components/script/test_init.py

```diff
@@ -446,3 +446,24 @@ def test_reload_service(self):
         "device-in-both",
         "device-in-last",
     }
+
+
+async def test_config(hass):
+    """Test passing info in config."""
+    assert await async_setup_component(
+        hass,
+        "script",
+        {
+            "script": {
+                "test_script": {
+                    "alias": "Script Name",
+                    "icon": "mdi:party",
+                    "sequence": [],
+                }
+            }
+        },
+    )
+
+    test_script = hass.states.get("script.test_script")
+    assert test_script.name == "Script Name"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5020228)